src/dso_api/dynamic_api/routers.py

```diff
@@ -323,11 +361,32 @@ def _build_openapi_views(self, datasets: Iterable[Dataset]) -> list[URLPattern]:
             dataset_id = dataset.schema.id
             results.append(
                 path(
-                    dataset.path + "/",
-                    get_openapi_json_view(dataset),
+                    "/" + dataset.path + "/",
```

Comment:
```suggestion
                    f"/{dataset.path}/",
```

en zou je hier niet de -noslash als default aanbieden?